src/test/scala/mesosphere/marathon/MarathonSchedulerActorTest.scala

```diff
@@ -339,8 +339,16 @@ class MarathonSchedulerActorTest extends TestKit(ActorSystem("System"))
     val plan = DeploymentPlan("foo", origGroup, targetGroup, List(DeploymentStep(List(StopApplication(app)))), Timestamp.now())
 
     when(tracker.get(app.id)).thenReturn(Set(taskA))
+    when(repo.store(any())).thenReturn(Future.successful(app))
```

Comment:
That test only checks if the rate limiter gets reset. I don't think we should check the instance count here.